### ISSUE TITLE
In der Nähe: Restaurants und Gaststätten

### ISSUE THREAD

**Issue by mojoaxel** [fuerthwiki/wiki#67]
Evtl. sollten wir eine Kategorie "Restaurants und Gaststätten" in die Nearby-Funktion integrieren?

**Comment by KasaFue** [comment]
Ich fände es gut, allerdings wird hier bei uns vermutlich nicht allzuviel angezeigt, da wir auf dem "Auge etwas Blind" sind - sprich die aktuellen Kneipen etc. sind bei uns unterdurchschnittlich vertreten. Aber vielleicht erhöht es ja die Motivation einzelner, sich dieser Sache anzunehmen, wenn die Funktion vorhanden ist  ;-)

**Comment by markmuzenhardt** [comment]
Ja, unbedingt. Dafür werde ich das Gaststätten-Formular demnächst auf Vordermann bringen und die nötigen Daten bereitstellen.

**Comment by FxGsm** [comment]
Hatten dazu ja schon mal eine Liste mit den gewünschten Kriterien zusammen getragen:
Anschrift,
E-Mailadresse,
Internetseite,
„vorher“, 
Besonderheiten, 
Telefonnummer

Genauso würde das Eröffnungsjahr die Chance bieten, bei der Abfrage nach „neu in der Stadt“ suchen zu lassen (das System errechnet dann welche Lokale es erst … sagen wir mal ein Jahr lang … gibt und berücksichtigt in der Suche nur diese, wenn das „neu in der Stadt“ angekreuzt wäre). Ein Nice to have, nichts wichtiges.

Küche: Auswahl vorgegebener Punkte wie „Gut bürgerlich“, Italienisch“ usw. ergänzbar durch Handeingabe.
Spezialitäten: Auswahl wie Küche mit vorgegebenen Punkten, ergänzbar um Handeingabe.
Vegetarische Speisen: Ja / Nein / leer=Keine Info
Vegane Speisen: Ja / Nein / leer=Keine Info
Speisekarte online: Ja / Nein / leer
Biere: Angabe von welchen Brauereien (gibt Lokale mit mehreren)
Fassbier: Ja/Nein ggf als Checkbox
Außenbereich Ja / Nein ggf als Checkbox, plus Auswahl ob Biergarten, an der Straße, Terrasse … den Suchenden interessiert es ja erstmal ob er überhaupt draußen sitzen kann. Dann jedoch gibt es jenseits davon die Geschmacksfrage ob es ok ist an der Straße zu sitzen ;-)
Kinderfreundlich: Ja / Nein / leer=keine Info
Barrierrefrei: Ja / Nein / leer=Keine Info
Nebenzimmer: Ja / Nein leer=keine Info
Sitzplätze innen: Anzahl
Sitzplätze außen: Anzahl
Öffnungszeiten - Wichtig um Abfragen zu ermöglichen, à la: Ich suche Lokal das morgen offen hat, bzw. das heute offen hat und will auch nur diese angezeigt bekommen.
Zahlungsmöglichkeit: Bar und/oder EC 

Das ist keine Insel-Spielerei, vielmehr bin ich der Überzeugung, dass eine gute Formular- und Suchfunktion für diesen Bereich dem FürthWiki eine große Hilfe ist in anderen gesellschaftlichen Bereichen als den historisch interessierten Fürthern Bekanntheit und Relevanz im Alltag zu erlangen. Quasi ein starkes Standbein in der Gegenwart.
